docs/source/driver/postgresql.rst

```diff
@@ -188,6 +188,129 @@ PostgreSQL allows defining new types at runtime, so the driver must
 build a mapping of available types.  This is currently done once at
 startup.
 
-Type support is currently limited.  Parameter binding and bulk
-ingestion support int16, int32, int64, and string.  Reading result
-sets is limited to int32, int64, float, double, and string.
+Type support is currently limited depending on the type and whether it is
+being read or written.
+
+.. list-table:: Arrow type to PostgreSQL type mapping
+   :header-rows: 1
+
+   * - Arrow Type
+     - As Bind Parameter
+     - In Bulk Ingestion
```

Comment:
What is exactly meant with the "In Bulk Ingestion" here? That column seems to be the resulting PostgreSQL type, right? 
Is there a difference with "As Bind Parameter"?

Reply:
Creating a Postgres table from an Arrow table uses a different path (going through COPY) than bind parameters so the types supported aren't necessarily the same

Reply:
But there is only one column of postgres types, so currently the table doesn't mention which postgres types will be used through bind parameters, only if it is supported?

Reply:
(also, for the bind parameters, we do support passing the data as a pyarrow table/batch/stream. That can then also use different types?)

Reply:
Ah, I'm not sure what I was thinking. I can add the Postgres type for that as well. (Technically it gets resolved to an OID so it's not necessarily exactly that type name, I think, but would be the same.)

And I would expect them to be the same just that it may not be supported in one or the other

Reply:
https://github.com/apache/arrow-adbc/issues/1385

Reply:
> (also, for the bind parameters, we do support passing the data as a pyarrow table/batch/stream. That can then also use different types?)

No, whether you pass data as Python or PyArrow types, it all gets converted into PyArrow types

Reply:
OK, thanks for the clarification!